sklearn/feature_selection/from_model.py

```diff
@@ -183,10 +187,17 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
     `threshold_`: float
         The threshold value used for feature selection.
     """
-    def __init__(self, estimator, threshold=None, prefit=False):
+    def __init__(self, estimator, threshold=None, prefit=False, max_n_features=None):
```

Comment:
Is this line more than 79 characters long? Make sure your code follows PEP 8 standards.